service/src/test/resources/policy-set-with-one-policy-nocondition.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
Please create new policies needed for your tests to avoid one test affecting another.